### ISSUE TITLE
Are you going to continue?

### ISSUE THREAD

**Issue by ripSquid** [Iambian/tilink#1]
I have been trying to do this with serial2ti83 but on arduino.
It doesn't seem to work that good so I hope this would work better. :)
I am looking forward to seeing this being done sometime

**Comment by geekbozu** [comment]
We have been working on this as we have time...we have been focusing on some usb stuff related to the project which has made it slow going.
If you just want a serial link
https://github.com/geekbozu/PicoGreylink
Worked the last time I tested it...Iambian helped me with that variant.

**Comment by Iambian** [comment]
It's been in the works for some time, but the next step in the plan is building a USB MTP driver to give the RP2 a way to send/receive files between a TI-84+SE (or compatible) calculators and a PC in a user-friendly manner. Thing is, none of the libraries actually support it so much of the time spent is in trying to understand the details of MTP and the TinyUSB library. And USB in general since I started with zero experience in dealing with USB in any capacity other than an end user.

In short, yes I'm going to continue, but the current step in the process has many things before anything can be written, let alone committed and pushed.
